src/plugin/context.rs

```diff
@@ -339,8 +339,8 @@ impl RapierContext {
         }
     }
 
-    /// This method makes sure tha the rigid-body positions have been propagated to
-    /// their attached colliders, without having to perform a srimulation step.
+    /// This method makes sure that the rigid-body positions have been propagated to
```

Comment:
it's actually `simulation` step :sweat_smile:  ; I'll fix in another unrelated pr